spec/models/rubric_criterion_spec.rb

```diff
@@ -241,4 +241,29 @@
       end
     end
   end
+
+  context 'A rubric criteria with levels' do
+    before(:each) do
+      @criterion = create(:rubric_criterion)
+      @levels = @criterion.levels
```

Comment:
would it make sense to make this as an instance variable instead of continuously defining levels = @criterion.levels within each test block?

Reply:
that's fine with me